composer.json

```diff
@@ -24,7 +24,7 @@
         "elife/api-client": "^1.0@dev",
         "elife/api-sdk": "dev-master",
         "elife/civi-contacts": "dev-master",
-        "elife/patterns": "dev-master",
+        "elife/patterns": "dev-peer-review-process-section",
```

Comment:
This will need to be changed back to master

Reply:
I will change that before the merge to prod